src/lib/y2network/config_writer/sysconfig.rb

```diff
@@ -32,7 +36,11 @@ class Sysconfig
       # @param config [Y2Network::Config] Configuration to write
       def write(config)
         routes = config.routing.routes.map { |r| route_to_hash(r) }
-        Yast::Routing.Import("routes" => routes)
+        Yast::Routing.Import(
+          "ipv4_forward" => config.routing.forward_ipv4,
```

Comment:
Can `config.routing` be `nil?`?

Reply:
That is what asked above, if we should handle situations like that.

Reply:
Well, I do not like much to have nil checks everywhere. If we need something like this, then at least use NullObject pattern that behave reasonably for usual cases and handle only specific ones.